Cargo.toml

```diff
@@ -13,8 +13,9 @@ edition = "2018"
 
 [features]
 default = [ "std" ]
-std = [ "unicode-normalization", "serde/std" ]
+std = [  "serde/std", "alloc" ]
```

Comment:
Seems like we should enable "std" in `unicode-nomalization`, but it build without this.
```rust
std = [  "serde/std", "alloc", "unicode-normalization/std" ]
```

Reply:
6f967f1